autoload/lsp/buffer.vim

```diff
@@ -170,6 +176,7 @@ export def CurbufGetServerChecked(feature: string = null_string): dict<any>
     return {}
   endif
 
+  
```

Comment:
redundant blank line.

Reply:
i saw your commit, as i knew yegappan would not squash the commits when merging, maybe you need to squash it later, otherwise you would see the commits history (and tho the contribute count too) a little redundant too, :smile:
// just an irrelevant comment, and it's on yegappan.